### DISCUSSION TITLE
Is the funlen repo still maintained?

### DISCUSSION THREAD

**Discussion by martialblog** [golangci/golangci-lint#3855] [Q&A] [answered]
Hi, hope this is the right place for my question. 

I was just wondering if someone knows if the https://github.com/ultraware/funlen repository is still being maintained?

I opened a PR to fix/improve some things, but no feedback thus far. If not, I'd be willing to maintain the fork I created.

Cheers
Markus

**Comment by ldez** [comment]
Hi,

I recommend opening an issue on the funlen repository to ask this question.

**Reply by martialblog** [reply]
Hi, thanks for the reply. Unfortunately there seems to be little activity in the repo. From the comments it reads as if the original maintainer no longer has access to the repo.

I do think the `funlen` changes make sense to include (I do admit that I might be biased here... )

https://github.com/ultraware/funlen/pull/14

**Reply by ldez** [reply]
I still recommend opening a dedicated issue on the funlen repository to ask if the repo is maintained.
You can ping me on this new issue.

We will wait at least one month after that, if we don't have an answer we will create a fork.

**Reply by martialblog** [reply]
Sounds good!

**Reply by martialblog** [reply]
@ldez thanks for your support at the funlen repo. Let's see what happens. 

https://github.com/ultraware/funlen/issues/15

I can take care of maintaining the repo. I'd love to contribute to the golangci project, it's a great tool

Regards,
Markus

**Reply by ldez** [reply, marked as answer]
We have the answer now: https://github.com/ultraware/funlen/issues/15#issuecomment-1648025030
